acceptance/serverless/serverless.yml

```diff
@@ -0,0 +1,90 @@
+service: iopipe-python-serverless-acceptance-tests
```

Comment:
`iopipe-lib-master-acceptance-test` is node, so could we have this be `iopipe-python-master-acceptance-test`?

Reply:
@pselle Since the acceptance tests will be targeting more than just serverless I changed it to `iopipe-python-master-serverless-acceptance-tests`. Think the node.js service should be renamed to `iopipe-js` instead of `iopipe-lib` as well.

Reply:
I agree, should be `iopipe-js`

when you say targeting more than just serverless ... where is that happening?

Reply:
Not to get too deep in the naming hole (which is why I offered a suggestion) -- changing `python` to `py` could make the name shorter. I also had suggested `tests`=>`test` per the node naming (or make pluralizing it an issue on the node project, whichever)

Reply:
Saw your comment about `serverless` in Slack, adding future acceptance tests for other frameworks, especially since say, Zappa, is popular, is really cool :+1: